api/views.py

```diff
@@ -330,7 +330,16 @@ def endpoint(request):
                                                 fraction_result = directions.Result.objects.filter(issledovaniye=issled, fraction=fraction_rel.fraction).order_by("-pk")[0]
                                             else:
                                                 fraction_result = directions.Result(issledovaniye=issled, fraction=fraction_rel.fraction)
-                                            fraction_result.value = str(results[key]).strip()
+                                            tmp_replace_value = {}
+                                            if fraction_rel.replace_value:
+                                                try:
+                                                    tmp_replace_value = json.loads(fraction_rel.replace_value)
+                                                except Exception:
+                                                    tmp_replace_value = {}
+                                            if tmp_replace_value.get(results[key], None):
```

Comment:
Правильнее так будет:
```suggestion
                                            if str(results[key]).strip() in tmp_replace_value:
```